src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Dex2Hex.txt

```diff
@@ -0,0 +1,138 @@
+﻿// ******** INTEGER PARAMETERS WITHOUT PLACES ********
+
+>> Dec2Hex(0)
+"0"
+
+>> Dec2Hex(Blank())
+"0"
+
+>> Dec2Hex(Error())
+#Error(Kind=InvalidArgument)
+
+// Positive Integer Parameters
+>> Dec2Hex(165)
+"A5"
+
+>> Dec2Hex(3213)
+"C8D"
+
+>> Dec2Hex(549755813887)
+"7FFFFFFFFF"
+
+// Negative Integer Parameters
+>> Dec2Hex(-1)
+"FFFFFFFFFF"
+
+>> Dec2Hex(-165)
+"FFFFFFFF5B"
+
+>> Dec2Hex(-3213)
+"FFFFFFF373"
+
+>> Dec2Hex(-549755813888)
+"8000000000"
+
+// ******** DECIMAL PARAMETERS ********
+
+>> Dec2Hex(3.74)
+"3"
+
+>> Dec2Hex(-3.74)
+"FFFFFFFFFC"
+	
+
+// ******** INTEGER PARAMETERS WITH PLACES ********
+
+>> Dec2Hex(45780)
+"B2D4"
+
+>> Dec2Hex(45780, 3)
```

Comment:
>Dec2Hex(45780, 3) [](http://example.com/codeflow?start=3&length=17)

Another test with errors: `Dec2Hex(1/0, Sqrt(-1))`, result should be `#Error(Kind=Div0,Numeric)`